File: Programación/PrimerTrimestre/Boletin2/Ejercicio8.java
package Boletin2;

import java.util.Scanner;

public class Ejercicio8 {
	public static void main(String[] args) {
		Scanner teclado = new Scanner (System.in);
		System.out.print("Introduce un número: ");
		int numero = teclado.nextInt();
		
		for (int i=1; i<=numero; i++) {
			if(numero%i==0) {
				if(i!=numero)
					System.out.print(i + ", ");
				else
					System.out.println(i);
			}
		}
	}
}

File: Lenguaje de Marcas/TercerTrimestre/Boletin1/ejercicio8.js
let numero = prompt("Introduce un número: ")
let primo = true
for(let i=2; i<numero; i++) {
    if(numero%i==0)
        primo = false
}

if(primo == true)
    document.writeln("El número es primo")
else
    document.writeln("El número no es primo")